src/App/components/PageHeader/NetworkSelector/NetworkSelector.tsx

```diff
@@ -64,15 +64,27 @@ export default function NetworkSelector(props: propsIF) {
                     variants={ItemEnterAnimation}
                     tabIndex={0}
                 >
-                    <ChainNameStatus tabIndex={0}>
+                    <ChainNameStatus
+                        tabIndex={0}
+                        active={chain.chainId === chainId}
+                    >
                         <img
                             src={chain.logoUrl}
                             alt={chain.displayName}
                             width='21px'
                             height='21px'
                             style={{ borderRadius: '50%' }}
                         />
-                        {chain.displayName}
+                        <p
```

Comment:
Would prefer here a <Text> component from the common lib.  

`<Text color={chain.chainId === chainId && 'var(--accent1)'} />`